src/natcap/invest/fisheries/fisheries_io.py

```diff
@@ -14,11 +14,6 @@
 from .. import reporting
 from .. import utils
 
-try:
-    unicode
-except NameError:
-    # No unicode in Python 3
-    unicode = str
 
```

Comment:
Huh, I'm surprised we included this import in this file, despite the fact that `unicode` isn't used anywhere in this file!